src/libs/actions/Policy.js

```diff
@@ -244,51 +243,57 @@ function removeMembers(members, policyID) {
 }
 
 /**
- * Merges the passed in login into the specified policy
+ * Adds members to the specified workspace/policyID
  *
- * @param {Array<String>} logins
+ * @param {Array<String>} memberLogins
  * @param {String} welcomeNote
  * @param {String} policyID
  */
-function invite(logins, welcomeNote, policyID) {
-    Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`, {
-        alertMessage: '',
-    });
+function addMembersToWorkspace(memberLogins, welcomeNote, policyID) {
+    const membersListKey = `${ONYXKEYS.COLLECTION.POLICY_MEMBER_LIST}${policyID}`;
+    const logins = _.map(memberLogins, memberLogin => OptionsListUtils.addSMSDomainIfPhoneNumber(memberLogin));
 
-    // Optimistically add the user to the policy
-    const newEmployeeLogins = _.map(logins, login => OptionsListUtils.addSMSDomainIfPhoneNumber(login));
-    const employeeUpdate = {};
-    _.each(newEmployeeLogins, login => employeeUpdate[login] = {});
-    Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY_MEMBER_LIST}${policyID}`, employeeUpdate);
+    const optimisticData = [
+        {
+            onyxMethod: 'merge',
+            key: membersListKey,
 
-    // Make the API call to merge the login into the policy
-    DeprecatedAPI.Policy_Employees_Merge({
+            // Convert to object with each key containing {pendingAction: ‘add’}
+            value: _.object(logins, Array(logins.length).fill({pendingAction: 'add'})),
```

Comment:
Use the constant https://github.com/Expensify/App/blob/23f4fc80ca4a2df155a58dfc51eef301125a1fc3/src/CONST.js#L800